addons/l10n_es/data/account_fiscal_position_template_data.xml

```diff
@@ -14,18 +14,27 @@
         <!-- ************************************************************* -->
 
         <record id="fp_nacional" model="account.fiscal.position.template">
+            <field name="sequence">1</field>
             <field name="name">Régimen Nacional</field>
             <field name="chart_template_id" ref="account_chart_template_common"/>
+            <field name="auto_apply" eval="True"/>
+            <field name="country_id" ref="base.es"/>
         </record>
 
         <record id="fp_intra" model="account.fiscal.position.template">
+            <field name="sequence">2</field>
             <field name="name">Régimen Intracomunitario</field>
             <field name="chart_template_id" ref="account_chart_template_common"/>
+            <field name="auto_apply" eval="True"/>
+            <field name="country_group_id" ref="base.europe"/>
         </record>
 
+        <!-- TODO: verify with JCO -->
         <record id="fp_extra" model="account.fiscal.position.template">
+            <field name="sequence">3</field>
             <field name="name">Régimen Extracomunitario / Canarias, Ceuta y Melilla</field>
             <field name="chart_template_id" ref="account_chart_template_common"/>
+            <field name="auto_apply" eval="True"/>
```

Comment:
You can leave it like this for now.  If there are partners within Canarias, ... they can put it on the partner and we will check later for a full solution.

Reply:
Ok.